facebook/flipper
# Update to enable Mac universal builds of Flipper

## Summary

New M1 Mac's must use rosetta to use Flipper. The Electron builder only produces x86_64 builds and the PortForwardingApp is precompiled and committed in the code base with an x64 build. This PR adds universal building of the electron app and the PortForwardingMacApp

## Changelog

Enable universal mac (mac/x64) builds

## Test Plan

* `yarn build --mac` to build the application as per usual
* run the flipper.app as usual
* verify that app is working and that on an m1, the `Kind` is `Apple`
